src/Symfony/Component/Finder/Finder.php

```diff
@@ -153,6 +155,32 @@ public function date($dates)
         return $this;
     }
 
+    /**
+     * Adds rules that files must match like $finder->name() but with case-insensitivity set.
+     *
+     * You can use patterns (delimited with / sign), globs or simple strings.
+     *
+     *     $finder->iname('*.php')
+     *     $finder->iname('/\.php$/') // same as above
```

Comment:
As far as I understand it with the current implementation the two examples above are not fully equivalent (the glob will match `foo.PHP` while the regex wouldn't), right? Does it make sense to modify any passed regex by adding the `i` modifier?

Reply:
That's right the glob `$finder->iname('*.php')` will match `*.php` and `*.PHP`. The regex should be `/\.php$/i` to be the same. Will fix it.

Adding the `i` modifier to every regex passed to `$finder->iname()` makes sense to me. Will add it.